server_management/management/commands/_core.py

```diff
@@ -173,3 +173,11 @@ def run_tasks(env, tasks, user=None):
 
         # Check result
         check_request(task, task_result)
+
+
+def can_push():
+    try:
+        return settings.SERVER_MANAGEMENT_ENABLE_PUSHDB
+    except: # pylint: disable=bare-except
+        print('SERVER_MANAGEMENT_ENABLE_PUSHDB setting not found in base.py defaulting to False.')
+        return False
```

Comment:
This is unnecessarily complex - I'm not sure this needs a helper function at all.

`getattr(settings, 'SERVER_MANAGEMENT_ENABLE_PUSHDB', False)`